__tests__/app.test.js

```diff
@@ -37,4 +38,20 @@ describe("ANY /invalidpath", () => {
             expect(body.msg).toBe("invalid path")
         })
     })
+})
+
+describe("/api", () => {
+    test("200: GET respond with an object describing all available end points", () => {
+       return request(app)
```

Comment:
We could require the endpoints.json file in here and check against it instead of readFile 🙂